### PR TITLE
Chore: Updates for the code snippets

### PR DESCRIPTION
**Pull request by jonniebigodes**
This pull request addresses some of the feedback for the documentation:

What was done:
- The json/shell snippets are removed and their content is now shown in the markdown file.
- The writing docs snippets, the mdx examples are updated.

Feel free to provide feedback.

### CODE REVIEW

File: docs/snippets/common/badge-story.mdx.mdx [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 <!---Badge.stories.mdx --->
 
+<!--- This is your Story template function, shown here in React -->
```
Comment (tmeasday) [inline docs/snippets/common/badge-story.mdx.mdx:5]: Can we put this above the `export const Template = ` line?

File: docs/snippets/common/checkbox-story.mdx.mdx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ```md
 <!--- Checkbox.stories.mdx -->
 
+<!--- This is your Story template function, shown here in React -->
```
Comment (tmeasday) [inline docs/snippets/common/checkbox-story.mdx.mdx:4]: Can we put this above the `export const Template = ` line?

File: docs/snippets/common/component-story-mdx-story-by-name.mdx.mdx [hunk 3]
```diff
@@ -1,4 +1,8 @@
 ```md
+<!-- Button.stories.mdx --->
+
+<!--- This is your Story template function, shown here in React -->
```
Comment (tmeasday) [inline docs/snippets/common/component-story-mdx-story-by-name.mdx.mdx:4]: Can we put this above the `export const Template = ` line?

File: docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx [hunk 4]
```diff
@@ -1,4 +1,8 @@
 ```md
+<!-- MyComponent.stories.mdx --->
+
+<!--- This is your Story template function, shown here in React -->
```
Comment (tmeasday) [inline docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx:4]: Can we put this above the `export const Template = ` line?

**Review by tmeasday — APPROVED** [review]
Looks good, let's move a couple comments

File: docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx (hunk 4 above)
Comment (jonniebigodes) [reply docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx:4]: Sure, i'll update it shortly for all the snippets.

File: docs/snippets/common/badge-story.mdx.mdx (hunk 1 above)
Comment (jonniebigodes) [reply docs/snippets/common/badge-story.mdx.mdx]: Updated with the latest commit

File: docs/snippets/common/checkbox-story.mdx.mdx (hunk 2 above)
Comment (jonniebigodes) [reply docs/snippets/common/checkbox-story.mdx.mdx]: Updated with the latest commit

File: docs/snippets/common/component-story-mdx-story-by-name.mdx.mdx (hunk 3 above)
Comment (jonniebigodes) [reply docs/snippets/common/component-story-mdx-story-by-name.mdx.mdx]: Updated with the latest commit

File: docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx (hunk 4 above)
Comment (jonniebigodes) [reply docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx]: Updated with the latest commit

**Pull request merged**